xlators/mount/fuse/src/fuse-bridge.c

```diff
@@ -6177,7 +6183,7 @@ fuse_thread_proc(void *data)
         iov_in[1].iov_base = iobuf->ptr;
 
         iov_in[0].iov_len = msg0_size;
-        iov_in[1].iov_len = psize;
```

Comment:
Do you still need psize?

Reply:
> Do you still need psize?

Thanks, indeed no longer need for psize. This draft needs further improvements and is still under development and test.

I will push my updates as soon as possible.